libraries/AP_Math/crc.cpp

```diff
@@ -578,3 +578,14 @@ uint8_t parity(uint8_t byte)
 
     return p;
 }
+
+// sums the bytes in the supplied buffer, returns that sum mod 256
+// (i.e. shoved into a uint8_t)
+uint8_t sum_of_bytes_in_buffer_mod_256(uint8_t *data, uint16_t count)
```

Comment:
Wanna just call it `sum8`?
https://stackoverflow.com/questions/71162153/checksums-sum8-sum16-sum24-sum32-fletcher4-fletcher8-and-so-on-original-impl